net-mgmt/collectd/src/opnsense/mvc/app/models/OPNsense/Collectd/General.xml

```diff
@@ -113,5 +113,29 @@
             <default>1</default>
```

Comment:
^^^ model version number was not bumped, see above

Reply:
bumped to 1.1.0